src/OrleansTestKit/OrleansTestKit.csproj

```diff
@@ -24,9 +24,9 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Orleans.Reminders" Version="7.2.0" />
-    <PackageReference Include="Microsoft.Orleans.Streaming" Version="7.2.0" />
-    <PackageReference Include="Microsoft.Orleans.Runtime" Version="7.2.0" />
+    <PackageReference Include="Microsoft.Orleans.Reminders" Version="7.*" />
+    <PackageReference Include="Microsoft.Orleans.Streaming" Version="7.*" />
+    <PackageReference Include="Microsoft.Orleans.Runtime" Version="7.*" />
```

Comment:
Does the library run correctly against a project that targets a version < 7.2.0? The binary incompatibility of the grain interface may make this a hard requirement. It's been a while since I've dealt with the binary compatibility rules of .NET... a project explicitly targeting it with tests may be required to confirm?